### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by Mervill**
I wrote a script to find duplicate property assignments in atom definitions. Here's what it turned up. I removed duplicates on the assumption that the lowest copy of the value became its actual game value.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -126,7 +126,6 @@
 
 /obj/item/weapon/storage/box/syringes
 	name = "box of syringes"
-	desc = "A box full of syringes."
 	desc = "A biohazard alert warning is printed on the box."
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/weapons/storage/boxes.dm]: can you remove 130 instead?

File: code/modules/clothing/spacesuits/syndi.dm [hunk 2]
```diff
@@ -4,7 +4,6 @@
 	desc = "Top secret spess helmet."
 	icon_state = "syndicate"
 	item_state = "syndicate"
-	desc = "Has a tag: Totally not property of an enemy corporation, honest."
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/spacesuits/syndi.dm]: remove 4 instead.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/jobs.dm [hunk 3]
```diff
@@ -2,8 +2,6 @@
 //Chef
 /obj/item/clothing/head/chefhat
 	name = "chef's hat"
-	desc = "It's a hat used by chefs to keep hair out of your food. Judging by the food in the mess, they don't work."
-	icon_state = "chef"
```
Comment (ChangelingRain) [inline code/modules/clothing/head/jobs.dm]: This isn't duplicated?

File: code/modules/clothing/head/jobs.dm [hunk 4]
```diff
@@ -2,8 +2,6 @@
 //Chef
 /obj/item/clothing/head/chefhat
 	name = "chef's hat"
-	desc = "It's a hat used by chefs to keep hair out of your food. Judging by the food in the mess, they don't work."
-	icon_state = "chef"
 	item_state = "chef"
 	desc = "The commander in chef's head wear."
```
Comment (ChangelingRain) [inline code/modules/clothing/head/jobs.dm]: This seems more likely to be the duplicate/incorrect desc

**Comment by AnturK** [comment]
Add the script to the travis setup if it doesn't give any false-positives.

File: code/modules/clothing/head/jobs.dm (hunk 4 above)
Comment (Mervill) [inline code/modules/clothing/head/jobs.dm]: Not a fan of puns?

**Comment by optimumtact** [comment]
@AnturK probably better suited for another PR I'd say, but yes would be good to see

File: code/modules/clothing/head/jobs.dm (hunk 4 above)
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/head/jobs.dm]: this is funnier than it should be 😠

File: code/modules/mining/minebot.dm [hunk 5]
```diff
@@ -9,7 +9,6 @@
 	icon_state = "mining_drone"
 	icon_living = "mining_drone"
 	status_flags = CANSTUN|CANWEAKEN|CANPUSH
-	stop_automated_movement_when_pulled = 1
```
Comment (phil235) [inline code/modules/mining/minebot.dm]: this is very much needed. Apparently when you have /type1/type2/type3 and the following variable settings:
type1/var/myvar = 0 
type2/myvar = 1
type3/myvar = 0

Then the last one counts as a duplicate despite overriding the value in type2.

**Comment by phil235** [comment]
@optimumtact Why did you merge this before the 24hrs mark, and why did you not check the code for errors!

File: code/modules/mining/minebot.dm (hunk 5 above)
Comment (AnturK) [inline code/modules/mining/minebot.dm]: Not sure what you mean here ? There's simply the same value set twice on this type ?

Comment (RemieRichards) [inline code/modules/mining/minebot.dm]: yeah @phil235 scroll down.

Comment (phil235) [inline code/modules/mining/minebot.dm]: Welp, i'm stupid...

Comment (RemieRichards) [inline code/modules/mining/minebot.dm]: Nah, it wasn't in the diff, you had to pop open the rest of the file to notice.

File: code/modules/clothing/head/jobs.dm (hunk 3 above)
Comment (PKPenguin321) [inline code/modules/clothing/head/jobs.dm]: you were right

**Pull request merged**